src/main/java/com/minecolonies/coremod/colony/buildings/AbstractBuilding.java

```diff
@@ -543,14 +547,23 @@ public void onDestroyed()
 
         if(tileEntityNew != null)
         {
-            InventoryHelper.dropInventoryItems(world, this.location, (IInventory) tileEntityNew);
+            InventoryUtils.dropItemHandlerItems(world, this.location.getX(), this.location.getY(), this.location.getZ(), tileEntityNew.getItemHandler());
             world.updateComparatorOutputLevel(this.location, block);
             ConstructionTapeHelper.removeConstructionTape(this, world);
         }
         ConstructionTapeHelper.removeConstructionTape(this, world);
     }
 
     /**
+     * Returns the item handler that belongs to the colony building.
+     *
+     * @return {@link IItemHandler} item handler of the building.
+     */
+    public IItemHandler getTileItemHandler() {
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png 'Severity: MINOR') Move this left curly brace to the beginning of next line of code. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://home.kk-sc.de:9000/coding_rules#rule_key=squid%3ALeftCurlyBraceStartLineCheck)